src/main

```diff
@@ -1,9 +1,47 @@
-puts 'Hello!'
-require_relative './person'
-require_relative './decorator'
-person = Person.new(22, 'maximilianus')
-puts person.correct_name
-capitalized_person = CapitalizedDecorator.new(person)
-puts capitalized_person.correct_name
-capitalized_trimmed_person = TrimmerDecorator.new(capitalized_person)
-puts capitalized_trimmed_person.correct_name
+#!/usr/bin/env ruby
+require_relative 'app'
+
+def resolve_choice(choice); end
+
+def user_choice # rubocop:disable Metrics/CyclomaticComplexity, Metrics/MethodLength
+  app = App.new
+  flag = true
+  puts 'Welcome to School Library App!'
+  while flag
+    puts 'Please select an option by entering a number:'
+    puts '1. List all books'
+    puts '2. List all people'
+    puts '3. Create a person'
+    puts '4. Create a book'
+    puts '5. Create a rental'
+    puts '6. List all rentals for a given person id'
+    puts '7. Exit'
+    choice = gets.chomp.to_i
+    flag = false if choice == 7
+
+    case choice
+    when 1
+      app.list_all_books
+    when 2
+      app.list_all_people
+    when 3
+      app.create_person
+    when 4
+      app.create_book
+    when 5
+      app.create_rental
+    when 6
+      app.list_all_rentals_for_a_given_person_id
+    when 7
+      puts 'Thank you for using School Library App!'
+    else
+      puts 'Invalid option, please try again!'
+    end
+  end
+end
+
+def main
+  user_choice
+end
+
+main
```

Comment:
- [ ] Kindly add the file extension so that your file is named `main.rb` and not just `main`.
- [ ] Also kindly move all your ruby files or at least the `main.rb` file to the root directory because I had to `cd` into the `src` directory before running your app which should not be the case.
- Your app:
![Annotation 2022-06-17 061643](https://user-images.githubusercontent.com/85113567/174230142-ae7884fa-304d-40df-891d-0061d6d8e0a1.jpg)